### PR TITLE
add path to expressiveConfigPath

### PR DESCRIPTION
**Pull request by lowtower**
Add the project path to the `expressiveConfigPath`
Solves issue #30

### CODE REVIEW

**Comment by weierophinney** [comment]
@lowtower Can you please add a test for this? That will ensure that we don't back out the change later, re-introducing the problem you've reported in #30.

Thanks in advance!

**Comment by lowtower** [comment]
Ummh,
I have never done that before.
Nevertheless, I will have a look and try my best.
Cheers,
LT.

**Comment by lowtower** [comment]
Hello Matthew,
the problem lies in the "emptyness" of the variable `projectDir`.
This variable is not used at all. I guess it was implemented for a future additional argument for the project path.
I don't have a clue on how to test that.
As far as I understand, In the currently available tests a projectDir with the name 'project' is assumed/defined, which is not empty.
So, I would need a different `setUp` method.
Am I right?
Sorry for my stupidity (I am not a professional programmer).
Cheers,
LT.

**Comment by lowtower** [comment]
Hello @weierophinney ,
I am sorry for introducing code that makes the tests fail!
I am afraid that's got all to do with projectDir, because my changes are all very small.
A single `empty()` check makes the tests fail!
I am confused!
I would like to help, but I am afraid this is beyond my (very limited) capabilities.
Sorry,
LT.

**Comment by lowtower** [comment]
More than that I'm slipshod :-(

File: composer.json [hunk 1]
```diff
@@ -18,11 +18,11 @@
         "bin/zf-development-mode"
     ],
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^7.0 || ^7.1"
```
Comment (xtreamwayz) [inline composer.json]: Why did you remove PHP 5.6 support? This is a hotfix for a bug.

File: test/DisableTest.php [hunk 2]
```diff
@@ -30,15 +33,15 @@ class DisableTest extends TestCase
 
     public function setUp()
     {
-        $this->projectDir = vfsStream::setup('project', null, [
+        $this->projectDir = vfsStream::setup($this->projectDirName, null, [
```
Comment (xtreamwayz) [inline test/DisableTest.php]: Why are you changing all the tests by adding by moving the `project` into a property?

Comment (lowtower) [inline test/DisableTest.php]: This was meant to be a preliminary action before varying the projectDir, which seems to be the problem.
An empty projectDir causes the problem as far as I understand.
I asked Matthew, how to test against that as the projectDir is hardcoded to "project" in the tests.
I also "cleaned up" the code to make use of the same syntax for vfsStream.
I can revert all the changes  if necessary - still, I don't know how to test against an empty projectDir!

File: composer.json (hunk 1 above)
Comment (lowtower) [inline composer.json]: Sorry,
pushed to wrong repository :-(
I'm quite new to all this stuff.
reverted with https://github.com/zfcampus/zf-development-mode/pull/31/commits/f6526bb1a011df9ce3cf901e8924c972f1ff1ee1

**Comment by lowtower** [comment]
Closed with the help of @xtreamwayz 
Discussion is in [issue #30](https://github.com/zfcampus/zf-development-mode/issues/30)
